test/uk/gov/hmrc/tai/model/domain/EmploymentBuilderSpec.scala

```diff
@@ -229,8 +229,7 @@ class EmploymentBuilderSpec extends PlaySpec with MockitoSugar {
             LocalDate.parse("2017-07-24"),
             None,
             List(
-              AnnualAccount("taxDistrict1-payeRefemployer1", TaxYear(2017), Available, Nil, Nil),
-              AnnualAccount("taxDistrict1-payeRefemployer1-payrollNo88", TaxYear(2017), Available, Nil, Nil)
+              AnnualAccount(1, TaxYear(2017), Available, Nil, Nil)
```

Comment:
Should there be 2 AnnualAccounts here or is that no longer possible?

Reply:
Employment can have only one AnnualAccount with matching sequence number. We have considered "taxDistrict1-payeRefemployer1" and "taxDistrict1-payeRefemployer1-payrollNo88" as 2 different sequence now as 0 and 1. And Employment with sequenceNumber 1 is matching only with  AnnualAccount with sequenceNumber1.